src/context-providers/yup/custom-conditions/index.ts

```diff
@@ -41,3 +43,38 @@ YupHelper.addCondition("mixed", "equalsField", (values: unknown[], matches: unkn
 			return isEqual(values, fn.parent[`${matches}`]);
 	}
 });
+
+YupHelper.addCondition(
```

Comment:
can we have a new set of conditional rendering stories for dates?

Reply:
I think you missed this comment? I will like to have unit tests and stories for this too?

you can have a new set of conditional rendering stories for dates, can refer to `conditional-rendering-array.stories.tsx`.

add unit tests for this conditional rendering rule to `conditional-renderer.spec.tsx`